host/src/main/java/org/openjdk/skara/host/github/GitHubPullRequest.java

```diff
@@ -212,6 +212,11 @@ public String getTitle() {
         return json.get("title").asString();
     }
 
+    @Override
+    public void setTitle(String title) {
+        throw new RuntimeException("not implemented yet");
```

Comment:
Won't this be an issue if we run the unit tests as integration tests against real hosts?

Reply:
You are quite right, I was planning to save it for another task, as there are already existing issues with the real host tests..

Reply:
Ah ok, lets tackle it later then :+1: